### PR TITLE
Add page for new bitbucket bug

### PR DESCRIPTION
**Pull request by pyromaniackeca**
I tried finding a issue related to this on Bitbucket, but couldn't.
There are 10.5k+ issues so I didn't search all of them.

### CODE REVIEW

File: source/docs/building-new-bitbucket-branch-causes-authentication-error.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Building a new Bitbucket branch causes authentication error
+category: Build troubleshooting
+---
+
+When a branch with no new commits is added to Semaphore it refuses to build
```
Comment (dunjardl) [inline source/docs/building-new-bitbucket-branch-causes-authentication-error.md]: Semaphore, it

File: source/docs/building-new-bitbucket-branch-causes-authentication-error.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+and asks for a Bitbucket account reauthentication, even though authentication
```
Comment (dunjardl) [inline source/docs/building-new-bitbucket-branch-causes-authentication-error.md]: reauthentication. This takes place even though authentication

File: source/docs/building-new-bitbucket-branch-causes-authentication-error.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+is not the issue  and reauthentication does not fix the problem.
```
Comment (dunjardl) [inline source/docs/building-new-bitbucket-branch-causes-authentication-error.md]: issue and

File: source/docs/building-new-bitbucket-branch-causes-authentication-error.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+When new commits are pushed to the same branch Semaphore will build it without
```
Comment (dunjardl) [inline source/docs/building-new-bitbucket-branch-causes-authentication-error.md]: pushed to the same branch, Semaphore will

File: source/docs/building-new-bitbucket-branch-causes-authentication-error.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+issues.
```
Comment (dunjardl) [inline source/docs/building-new-bitbucket-branch-causes-authentication-error.md]: any issues.

**Comment by pyromaniackeca** [comment]
@dunjardl Thanks, fixed.

**Pull request merged**